routes/entryRoutes.js

```diff
@@ -131,14 +131,14 @@ routes.GETcohortEntries = function(req, res) {
   Student.find({program:cohort}, function(err, students){
     var studentIds = students.map(function(student){return student._id});
     console.log('studentIds: '+studentIds);
-    FormDB.find({_studentID:{$in:studentIds}}, function(err, forms){
-      forms.reduce(function(prev,form){
+    FormDB.find({_studentID:{$in:studentIds}}, function(err, forms) {
+      forms.forEach(function(form) {
```

Comment:
I would prefer to keep the reduce thing since Shruti and I ran into a lot of weird errors with loops (including for each) and the database calls